docs/source/user_guide.rst

```diff
@@ -151,7 +167,6 @@ EinsteinPy also supports smbolic calculations in :py:class:`~einsteinpy.utils.ch
 
 Future Plans
 ============
```

Comment:
blank line was required here I guess

Reply:
try to figure out why docs are not building https://circleci.com/gh/einsteinpy/einsteinpy/3278?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link
I am a little preoccupied :)

Reply:
oh, okay!